native/src/widget/slider.rs

```diff
@@ -60,6 +62,7 @@ impl<'a, Message, Renderer: self::Renderer> Slider<'a, Message, Renderer> {
     pub fn new<F>(
         state: &'a mut State,
         range: RangeInclusive<f32>,
+        step: f32,
```

Comment:
I think we should configure this in a `step` method instead of asking for it here. We can initialize it to `0.01` for backwards compatibility.

Reply:
Do you mean a method with signature `step(&mut self, step: f32)`? Or should it be `step(self, step: f32) -> Self`?